hazelcast/src/main/java/com/hazelcast/internal/dynamicconfig/ClusterWideConfigurationService.java

```diff
@@ -558,12 +557,13 @@ public Merger(NodeEngine nodeEngine, Operation replicationOperation) {
 
         @Override
         public void run() {
-            OperationService operationService = nodeEngine.getOperationService();
-            BinaryOperationFactory operationFactory = new BinaryOperationFactory(replicationOperation, nodeEngine);
             try {
-                // intentionally targeting partitions and not members to avoid races
-                // it has generates extra load, but it's worth it.
```

Comment:
what about races as in this comment?

Reply:
The race between a map operation that may trigger creation of a map container with a default config on a member before it becomes aware of dynamic config via split-brain merge also exists in the previous implementation (and still exists after this PR). I think this is a generic reference to the first design of `ClusterWideConfigurationService`, before pre-join operation support was introduced (which eventually replaced the original migration-based design as a means for joining members to become aware of dynamic config).